src/SyncTrayzor/Pages/SettingsView.xaml

```diff
@@ -108,8 +109,8 @@
                 </DockPanel>
             </GroupBox>
         </Expander>
-        
-        <StackPanel DockPanel.Dock="Bottom" Margin="0,10,0,0" Orientation="Horizontal" HorizontalAlignment="Right">
+
+        <StackPanel DockPanel.Dock="Bottom" Margin="0,10,0,0" Orientation="Horizontal" Height="37" HorizontalAlignment="Right" Width="204" VerticalAlignment="Bottom">
```

Comment:
Nothing except the vertical alignment, I will modify it and see what the best fixed solution would be!